schedule/functional/extra_tests_clamav.yaml

```diff
@@ -7,4 +7,4 @@ schedule:
     - installation/bootloader_start
     - boot/boot_to_desktop
     - console/clamav
-    - console/coredump_collect
+
```

Comment:
@pablo-herranz the ci is failing because of this extra line I think

Reply:
Line removed, let's see now.